solidity/contracts/converter/BancorConverter.sol

```diff
@@ -44,8 +44,8 @@ contract BancorConverter is IBancorConverter, TokenHandler, SmartTokenController
     IWhitelist public conversionWhitelist;          // whitelist contract with list of addresses that are allowed to use the converter
     IERC20Token[] public reserveTokens;             // ERC20 standard token addresses (prior version 17, use 'connectorTokens' instead)
     mapping (address => Reserve) public reserves;   // reserve token addresses -> reserve data (prior version 17, use 'connectors' instead)
-    uint32 public totalReserveRatio = 0;            // total ratio of all reservces, also used to efficiently prevent increasing
-                                                    // the total reserve ratio above 100%
+    uint32 public reserveRatio = 0;                 // total reserve weights, ratio between the reserves and the market cap, also used to
+                                                    // efficiently prevent increasing the total reserve weights above 100%

```

Comment:
@yudilevi : Please consider removing the `, also used to efficiently prevent increasing the total reserve weights above 100%` part of the comment.

Reply:
updated